### ISSUE TITLE
Couldn't create build

### ISSUE THREAD

**Issue by pmolina** [MercadoTrack/extension-local#7]
1. ```~$ git clone https://github.com/GMaiolo/mercado-track.git```
2. ```~$ cd mercado-track```
3. ```~/mercado-track$ npm run build```

Result:
```
> mercadotrack@0.2.1 build /Users/patriciomolina/mercado-track
> rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors

sh: webpack: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the mercadotrack@0.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/patriciomolina/.npm/_logs/2017-07-20T21_49_12_387Z-debug.log
```

Log:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/8.2.0_1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build' ]
2 info using npm@5.3.0
3 info using node@v8.2.0
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle mercadotrack@0.2.1~prebuild: mercadotrack@0.2.1
6 info lifecycle mercadotrack@0.2.1~build: mercadotrack@0.2.1
7 verbose lifecycle mercadotrack@0.2.1~build: unsafe-perm in lifecycle true
8 verbose lifecycle mercadotrack@0.2.1~build: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/patriciomolina/mercado-track/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/MacGPG2/bin:/Users/patriciomolina/work/bin
9 verbose lifecycle mercadotrack@0.2.1~build: CWD: /Users/patriciomolina/mercado-track
10 silly lifecycle mercadotrack@0.2.1~build: Args: [ '-c',
10 silly lifecycle   'rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors' ]
11 info lifecycle mercadotrack@0.2.1~build: Failed to exec build script
12 verbose stack Error: mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
12 verbose stack spawn ENOENT
12 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:33:16)
12 verbose stack     at emitTwo (events.js:125:13)
12 verbose stack     at ChildProcess.emit (events.js:213:7)
12 verbose stack     at maybeClose (internal/child_process.js:921:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
13 verbose pkgid mercadotrack@0.2.1
14 verbose cwd /Users/patriciomolina/mercado-track
15 verbose Darwin 16.5.0
16 verbose argv "/usr/local/Cellar/node/8.2.0_1/bin/node" "/usr/local/bin/npm" "run" "build"
17 verbose node v8.2.0
18 verbose npm  v5.3.0
19 error file sh
20 error code ELIFECYCLE
21 error errno ENOENT
22 error syscall spawn
23 error mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
23 error spawn ENOENT
24 error Failed at the mercadotrack@0.2.1 build script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 1, true ]
```

**Comment by GMaiolo** [comment]
Run `npm i webpack -g` and try again, probably missed that in the steps

**Comment by GMaiolo** [comment]
Updated the README with the correct steps

**Comment by pmolina** [comment]
I've executed ```npm run build``` after ```npm install```:

```
~/mercado-track$ npm run build

> mercadotrack@0.2.1 build /Users/patriciomolina/mercado-track
> rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors

Hash: 1c41a95ab500916d07d4
Version: webpack 2.7.0
Time: 3886ms
                   Asset       Size  Chunks                    Chunk Names
       content.bundle.js     844 kB       0  [emitted]  [big]  content
         popup.bundle.js     942 kB       1  [emitted]  [big]  popup
    background.bundle.js    10.3 kB       2  [emitted]         background
   content.bundle.js.map    1.07 MB       0  [emitted]         content
     popup.bundle.js.map    1.17 MB       1  [emitted]         popup
background.bundle.js.map    11.3 kB       2  [emitted]         background
           ../popup.html    1.89 kB          [emitted]
  ../images/icon_128.png    4.28 kB          [emitted]
        ../manifest.json  756 bytes          [emitted]
              ../LICENSE    1.07 kB          [emitted]
   [1] ./src/scripts/modules/utils/utils.js 2.33 kB {0} {1} {2} [built]
   [2] ./src/scripts/modules/item.model.js 1.32 kB {0} {1} {2} [built]
   [3] ./src/scripts/modules/storage.js 2.25 kB {0} {1} {2} [built]
   [5] ./src/scripts/modules/utils/dom.utils.js 2.05 kB {0} [built]
   [6] ./~/vue/dist/vue.common.js 403 kB {1} [built]
 [122] ./src/popup/store.js 500 bytes {1} [built]
 [123] ./src/scripts/modules/graph.js 502 bytes {0} [built]
 [124] ./~/materialize-css/bin/materialize.js 149 kB {1} [built]
 [125] ./src/popup/popup.sass 1.15 kB {1} [built]
 [175] ./src/popup/popup.js 1.24 kB {1} [built]
 [176] ./src/scripts/background.js 419 bytes {2} [built]
 [177] ./src/scripts/content.js 1.28 kB {0} [built]
 [178] ./~/hammerjs/hammer.js 73.8 kB {1} [built]
 [182] ./~/process/browser.js 5.42 kB {1} [built]
 [185] ./~/vuex/dist/vuex.js 18.5 kB {1} [built]
    + 174 hidden modules

ERROR in content.bundle.js from UglifyJs
Unexpected token: punc (() [content.bundle.js:4547,15]

ERROR in popup.bundle.js from UglifyJs
Unexpected token: punc (() [popup.bundle.js:78,15]

ERROR in background.bundle.js from UglifyJs
Unexpected token: punc (() [background.bundle.js:78,15]
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the mercadotrack@0.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/patriciomolina/.npm/_logs/2017-07-21T01_20_52_055Z-debug.log
```

Log:

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/Cellar/node/8.2.0_1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'run',
1 verbose cli   'build' ]
2 info using npm@5.3.0
3 info using node@v8.2.0
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle mercadotrack@0.2.1~prebuild: mercadotrack@0.2.1
6 info lifecycle mercadotrack@0.2.1~build: mercadotrack@0.2.1
7 verbose lifecycle mercadotrack@0.2.1~build: unsafe-perm in lifecycle true
8 verbose lifecycle mercadotrack@0.2.1~build: PATH: /usr/local/lib/node_modules/npm/bin/node-gyp-bin:/Users/patriciomolina/mercado-track/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/usr/local/MacGPG2/bin:/Users/patriciomolina/work/bin
9 verbose lifecycle mercadotrack@0.2.1~build: CWD: /Users/patriciomolina/mercado-track
10 silly lifecycle mercadotrack@0.2.1~build: Args: [ '-c',
10 silly lifecycle   'rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors' ]
11 silly lifecycle mercadotrack@0.2.1~build: Returned: code: 2  signal: null
12 info lifecycle mercadotrack@0.2.1~build: Failed to exec build script
13 verbose stack Error: mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/lifecycle.js:289:16)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at EventEmitter.emit (events.js:213:7)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:40:14)
13 verbose stack     at emitTwo (events.js:125:13)
13 verbose stack     at ChildProcess.emit (events.js:213:7)
13 verbose stack     at maybeClose (internal/child_process.js:921:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
14 verbose pkgid mercadotrack@0.2.1
15 verbose cwd /Users/patriciomolina/mercado-track
16 verbose Darwin 16.5.0
17 verbose argv "/usr/local/Cellar/node/8.2.0_1/bin/node" "/usr/local/bin/npm" "run" "build"
18 verbose node v8.2.0
19 verbose npm  v5.3.0
20 error code ELIFECYCLE
21 error errno 2
22 error mercadotrack@0.2.1 build: `rm -rf dist && webpack --config ./webpack.config.prod.js --progress --colors`
22 error Exit status 2
23 error Failed at the mercadotrack@0.2.1 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]
```

**Comment by GMaiolo** [comment]
The problem is with the minification process, for a non-uglified build workaround you can delete the `dist` folder and run `webpack --progress --colors`.
That would leave the exact same output but without minification.

On a side note, there problem is with the UglifyJS Harmony branch, as it has to be used for ES6 minification but, apparently, it's not being correctly installed after a fresh project installation; I'll take a look later today for a fix.

**Comment by GMaiolo** [comment]
@pmolina should be fixed with d3558f6b4c2a17382b271cd84e6acf79b6498d2b.
